zebra-state/src/service/finalized_state/disk_format/tests/snapshots/empty_column_families@mainnet_0.snap

```diff
@@ -1,5 +1,6 @@
 ---
 source: zebra-state/src/service/finalized_state/disk_format/tests/snapshot.rs
+assertion_line: 154
```

Comment:
Nit: If you update your version of `cargo insta`, it won't add the assertion line any more.

Reply:
I updated `cargo insta`, then ran `cargo insta test --review`, but nothing changed. Should I remove the lines by hand?